apiserver/facades/agent/provisioner/provisioner.go

```diff
@@ -134,6 +134,8 @@ func NewProvisionerAPI(ctx facade.Context) (*ProvisionerAPI, error) {
 		return nil, errors.Annotate(err, "instantiating network config API")
 	}
 
+	controllerConfigGetter := ctx.ServiceFactory().ControllerConfig()
```

Comment:
We probably want to pull this out at the `register.go` file and just pass the service directly.